### PR TITLE
Adding BUILD_URL and move on top of report

### PR DESCRIPTION
**Pull request by aleksbykov**
Links to build and job should be stored in ENV variable ( by default jenkins sets such variables)

BUILD_URL=<link> JOB_URL=<link> ./microbenchmarking.py check --update-db --results-path ~/tmp/ --email-recipients email@recipients.com

### CODE REVIEW

File: sdcm/microbenchmarking.py [hunk 1]
```diff
@@ -409,6 +415,8 @@ def parse_args():
                        help="Comma separated email addresses list that will get the report")
     check.add_argument("--report-path", action="store", default="",
                        help="Save HTML generated results report to the file path before sending by email")
+    check.add_argument("--build-url", action="store", default="", help='Set link to Jenkins build URL')
+    check.add_argument("--job-url", action="store", default="", help='Set link to Jenkins job URL')
```
Comment (amoskong) [inline sdcm/microbenchmarking.py]: If we always get value from environment values, then we don't need to add those two arguments.

Comment (bentsi) [reply sdcm/microbenchmarking.py]: Agree with Amos. Please remove those.

File: sdcm/results_microbenchmark.html [hunk 2]
```diff
@@ -21,6 +21,11 @@ <h3>
         <span> , Hostname: </span>
         <span class="blue"> {{ hostname }}</span>
     </div>
+    <h3>Dashboard:</h3>
```
Comment (bentsi) [inline sdcm/results_microbenchmark.html]: `Dashboard` -> `Links`

File: sdcm/microbenchmarking.py [hunk 3]
```diff
@@ -369,7 +375,7 @@ def main(args):
         if args.before_date:
             mbra.exclude_before_date(args.before_date)
     if args.mode == 'check':
-        mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=args.email_recipients.split(","))
+        mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=args.email_recipients.split(","), build_url=args.build_url, job_url=args.job_url)
```
Comment (bentsi) [inline sdcm/microbenchmarking.py]: MicroBenchmarkingResultsAnalyzer shouldn't be aware of how to get the URLs, so:
`build_url=os.environ.get('BUILD_URL', ""), job_url=...`

File: sdcm/microbenchmarking.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/microbenchmarking.py]: Think about flexibility, if there is no Env variables. Ok. remove

File: sdcm/results_microbenchmark.html (hunk 2 above)
Comment (aleksbykov) [reply sdcm/results_microbenchmark.html]: Got it

File: sdcm/microbenchmarking.py (hunk 3 above)
Comment (aleksbykov) [reply sdcm/microbenchmarking.py]: removed

**Pull request merged**